app/models.py

```diff
@@ -13,6 +13,9 @@ class Plant(db.Model):
     root_depth = db.Column(db.Integer)
     annual = db.Column(db.String(50))
 
+    def get_all():
```

Comment:
Thanks for adding this back in here!  Not sure where it got lost, probably at a merge conflict or something.  Maybe that means we should have a little more robust testing in the future??  (But we're aware of testing for this project)